erpnext/hr/report/employee_leave_balance_summary/employee_leave_balance_summary.py

```diff
@@ -128,3 +135,16 @@ def get_department_leave_approver_map(department=None):
 		approvers.setdefault(k, []).append(v)
 
 	return approvers
+
+def get_total_leave_allocated_map(filters):
+	total_leave_allocated_map = frappe._dict()
+	total_leave_allocated = frappe.db.sql("SELECT SUM(total_leaves_allocated) as total_leave_allocated, employee, leave_type FROM `tabLeave Allocation` WHERE from_date >= '"+filters.get("from_date")+"' GROUP BY employee, leave_type", as_dict = 1)
```

Comment:
- Split the query in multilines
- Use `.format` instead of `'"+filters.get("from_date")+"'`

Reply:
done